### DISCUSSION TITLE
Spay/Neuter costs

### DISCUSSION THREAD

**Discussion by Pletoo** [sheltermanager/asm3#1419] [Q&A / Help] [answered]
What is the best way to add the amount for spay/neuter to an animal's record? Should I put it in as a "Cost" or should I set up a Medical Profile? Or something else? Is there a way to automatically add this when an animal is altered or does that need to be manually done? 

Also what is the difference between profile and name when setting up a medical profile?
![profile name](https://github.com/sheltermanager/asm3/assets/157258152/16091b80-5c96-48db-8b24-acf073ff6be8)

**Comment by Tetzor** [comment, marked as answer]
Hi,

You should set up a medical profile and add the amount as a cost. You do need to add the regimen and give it in the animal record in order for the cost to be recorded. 

The text you add as the profile is what you will see in the profile drop down when choosing the regimen you want to add for an animal, the name will be auto-populated from the selected profile. 

I have added a link below to the training video for this.

https://www.youtube.com/watch?v=FejguGjfd90

Kind regards,

Jon

**Reply by Pletoo** [reply]
Okay. Thank you.
I guess I was hoping that profile would be preset categories (like medicine or surgery or exam) that would pull up different forms (for example, surgery doesn't need the dosage and frequency fields.) Then name would be for customization.

So far everything else has been very customizable, organized and streamlined, so maybe I can hope for a better way for altering to be integrated into the medical record in the future but for now, I will use the method suggested. (Create profile with cost, add regimen, give regimen, enter neuter date.) 

Thanks again.
